secrets-provider-test-chart/templates/conjur-cert-config-map.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Values.secretsProviderEnvironment.conjurSslCertificateName }}
+data:
+  ssl-certificate: {{ .Values.conjurSslCertificateValue | quote }}
```

Comment:
why `conjurSslCertificateValue` is in the open?
Consider adding `conjur.SslCertificate.Value` and `conjur.SslCertificate.Name`